json/global.js

```diff
@@ -0,0 +1,43 @@
+const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+const PASSWORD_REGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+
+export const testUser = {
+  email: "test@codeit.com",
+  password: "codeit101",
+};
+
+/*email 형식인지 체크해주는 함수*/
+export function emailValidCheck(email_address){     
```

Comment:
해당 함수도 다른 함수들처럼 동사로 시작하면 좋겠습니다. 매개변수또한 일반적으로 camelCase로 작성하니 참고하셨으면 좋겠습니다.

```suggestion
export function checkEmailValid(emailAddress){     
```